include/chemfiles/files/NcFile.hpp

```diff
@@ -76,6 +76,15 @@ namespace nc {
         void add(count_t start, count_t count, std::vector<float> data);
     };
 
+    class NcDouble final: public NcVariable {
```

Comment:
Could you add some tests for this in `tests/files/netcdf-file.cpp`?

Reply:
Oh, I missed that there were some tests in place for variable types. Added tests for `NcDouble`.